op-e2e/op_geth.go

```diff
@@ -37,9 +37,15 @@ var (
 
 // OpGeth is an actor that functions as a l2 op-geth node
 // It provides useful functions for advancing and querying the chain
+
+type Node interface {
```

Comment:
Nit: I'm not sure of the benefit to defining this interface, vs re-using the `EthInstance` -- not a big deal though either way.

Reply:
Fixed